static/js/app.js

```diff
@@ -692,11 +700,21 @@ $(function() {
             var places = this.searchBox.getPlaces();
             if (places && places.length > 0) {
               var place = places[0];
+              this.createHighlightPoint(place.geometry.location.lat(), place.geometry.location.lng(), HIGHLIGHT_TYPE_USER_SEARCH);
               this.setCenterWithMarker(place.geometry.location);
               this.map.setZoom(INIT_ZOOM);
             }
          },
-         setCenterWithMarker: function(loc) {
+        createHighlightPoint : function(lat, lng, highlightPointType) {
+            if (isNaN(lat) || isNaN(lng) || isNaN(highlightPointType)) return;
+
+            Backbone.$.ajax({
```

Comment:
Use the `$.post` shortcut, see [here](https://github.com/hasadna/anyway/blob/master/static/js/app.js#L674).